src/main/java/uk/gov/pay/ledger/transaction/model/TransactionFactory.java

```diff
@@ -125,6 +125,12 @@ private Transaction createRefund(TransactionEntity entity) {
                     .withWalletType(safeGetAsString(refundPaymentDetails, "wallet"))
                     .build();
 
+            SettlementSummary settlementSummary = new SettlementSummary(
+                    safeGetAsDate(transactionDetails, "capture_submitted_date"),
+                    safeGetAsDate(transactionDetails, "captured_date"),
+                    entity.getPayoutEntity().map(payoutEntity -> payoutEntity.getPaidOutDate()).orElse(null)
```

Comment:
Can we build with only paid out date as we don't need to look for `capture_submitted_date` and `captured_date` for refund ?